build.gradle

```diff
@@ -11,6 +11,8 @@ repositories {
     maven { url = "https://maven.quiltmc.org/repository/release/" }
     maven { url = "https://maven.terraformersmc.com/releases/" }
     maven { url = "https://repo.sleeping.town/" }
+
+    flatDir { dirs 'lib' }
```

Comment:
what does this do?

Reply:
loads jar libraries from the lib folder. temporary until modmenu 1.21 goes on their maven

Reply:
very cool i will remember this trick